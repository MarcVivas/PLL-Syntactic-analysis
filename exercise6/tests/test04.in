ALPHABET => => b, a <=    // Error expected


// States definition
STATES =>  ->q0, STATES <=   // Error expected

TRANSITIONS =>
->q0[a]...->q0  // Error expected missing ';'
->q0[b]...q1->;
q1->[a]...q1->;
q1->[b]...q1->;
<=
