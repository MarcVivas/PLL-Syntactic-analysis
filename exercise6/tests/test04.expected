Error at line 1: syntax error
Error at line 5: syntax error
Transition error: Letter a is not in the alphabet.
Error at line 9: syntax error
Transition error: Letter a is not in the alphabet.
Transition error: Letter b is not in the alphabet.
No accepting states specified.
Start states: { ->q0 }
Accepting states:
Intermediate states:
Alphabet:
Transitions:




